src/pages/Detail/Detail.js

```diff
@@ -1,12 +1,111 @@
-import React from "react";
+import React from 'react';
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
 import './Detail.scss';
 
 const Detail = () => {
+  const params = useParams();
+  const productId = params.id;
```

Comment:
params객체의 id 값을 새로운 변수에 담는 것 보다 Router컴포넌트에서  path parameter이름을 수정 하는게 나아보입니다.
```suggestion
  const { productId }= useParams();

```

Reply:
수정 완료!